numpy/core/fromnumeric.py

```diff
@@ -3276,7 +3276,8 @@ def around(a, decimals=0, out=None):
     See Also
     --------
     ndarray.round : equivalent method
-
+    numpy.round: equivalent method
+    numpy.round_: equivalent method
```

Comment:
```suggestion
    round_ : equivalent method
```
This is a nit just to maintain consistency with the rest of the documentation.